galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
@afgane the --extras-var or ansible in general is setting the build-time envs. In this case it will copy the file into the proper place. `GALAXY_CONFIG_TOOL_DATA_TABLE_CONFIG_PATH` is for run-time ENVs. This is replacing the entry in `galaxy.ini`.